lib/routes/dw/utils.ts

```diff
@@ -0,0 +1,167 @@
+                case 'bs':
+                    return 'Foto';
+                case 'bg':
+                    return 'Снимка';
+                case 'zh':
+                    return '图像来源';
+                case 'zh-hant':
+                    return '圖片來源';
+                case 'hr':
+                    return 'Foto';
+                case 'fa-af':
+                    return 'عکس';
+                case 'en':
+                    return 'Image';
+                case 'fr':
+                    return 'Image';
+                case 'de':
+                    return 'Bild';
+                case 'el':
+                    return 'Εικόνα';
+                case 'ha':
+                    return 'Hoto';
+                case 'hi':
+                    return 'तस्वीर';
+                case 'id':
+                    return 'Foto';
+                case 'sw':
+                    return 'Picha';
+                case 'mk':
+                    return 'Фотографија';
+                case 'ps':
+                    return 'انځور';
+                case 'fa-ir':
+                    return 'عکس';
+                case 'pl':
+                    return 'Zdjęcie';
+                case 'pt-002':
+                    return 'Foto';
+                case 'pt-br':
+                    return 'Foto';
+                case 'ro':
+                    return 'Imagine';
+                case 'ru':
+                    return 'Фото';
+                case 'sr':
+                    return 'Foto';
+                case 'es':
+                    return 'Imagen';
+                case 'tr':
+                    return 'Fotoğraf';
+                case 'uk':
+                    return 'Фото';
+                case 'ur':
+                    return 'تصویر';
+                default:
+                    return 'Image';
+            }
+        default:
+            return word;
+    }
+};
+
+const processHtml = ($: CheerioAPI, contentLinks) => {
+    $('img').each((_, elem) => {
+        try {
+            const id = $(elem).attr('data-id');
+            const contentLink = contentLinks.find((item) => String(item.targetId) === id);
+            $(elem).attr({
+                title: contentLink?.name,
+                alt: contentLink?.description,
+                src: `https://static.dw.com/image/${id}_${formatId}.jpg`,
+            });
+        } catch {
+            // no-empty
+        }
+    });
+    $('video').each((_, elem) => {
+        try {
+            $(elem).attr('poster', $(elem).attr('data-posterurl'));
+        } catch {
+            // no-empty
+        }
+    });
+    $('iframe').each((_, elem) => {
+        try {
+            $(elem).attr('src', $(elem).attr('data-src'));
+        } catch {
+            // no-empty
+        }
+    });
+    $('svg').remove(); // svg will screw up in a lot of rss readers
+};
+
+const processContent = (item, content) => {
+    const $text = load(content.text);
+    processHtml($text, content.contentLinks);
```

Comment:
Images from some articles (e.g., https://www.dw.com/en/us-republicans-win-house-control-both-houses-of-congress/a-70773778) include a style of
```
padding-bottom: 56.25%; height: 0; max-height: 0;
```

which basically hide the image.

Reply:
I am not sure if I understand you correctly, but I added a line of code to remove all styles attribute from images, and I think this would fix all issues with this